adapters/sidekiq/lib/opentelemetry/adapters/sidekiq/middlewares/client/tracer_middleware.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+# Copyright 2020 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Adapters
+    module Sidekiq
+      module Middlewares
+        module Client
+          class TracerMiddleware
+            def call(worker_class, job, _queue, _redis_pool)
+              tracer.in_span(
+                worker_class,
```

Comment:
Using the job name here makes sense, but I'm not sure if the cardinality is too high here.  Maybe calling this `sidekiq_job` or something makes more sense while having the attributes contain the worker name.

Reply:
I think this is OK. We used `sidekiq_job`, and to my mind, that's not specific enough. Cardinality of `worker_class` should not be huge.